### DISCUSSION TITLE
Add outcome model (Cox (HR))

### DISCUSSION THREAD

**Discussion by Naeemkh** [NSAPH-Software/CausalGPS#121] [General]

**Comment by Naeemkh** [comment]
@daniellebraun, are we planning to add this feature?

**Comment by daniellebraun** [comment]
I think so @wxwx1993 ?

**Reply by Naeemkh** [reply]
Hi @wxwx1993, please add some details to this feature.

**Comment by Naeemkh** [comment]
Moving this into the discussions ...
